assets/js/blocks/cart-checkout/payment-methods/express-payment/style.scss

```diff
@@ -91,6 +91,11 @@ $border-radius: 5px;
 			width: 50%;
 		}
 
+		> li:first-child:nth-last-child(1) {
```

Comment:
Wouldn't [`:only-child`](https://developer.mozilla.org/en-US/docs/Web/CSS/:only-child) work?

Reply:
Great catch, @senadir! Given that `:only-child` has the same level of browser support than `:first-child` and `:nth-last-child`, I switched to `:only-child`.

https://caniuse.com/mdn-css_selectors_only-child:

<img width="1375" alt="Screen Shot 2022-01-21 at 12 15 48" src="https://user-images.githubusercontent.com/3323310/150470125-12b31639-ac53-4cdd-99a7-2b673ca85685.png">